proto/minder/v1/minder.proto

```diff
@@ -2232,10 +2431,22 @@ message RuleType {
     Definition def = 4;
 
     // description is the description of the rule type.
-    string description = 5;
+    string description = 5 [
+        (buf.validate.field).string = {
+            pattern: "[A-Za-z][-/.!?,:;[:word:] *#\\[\\]()`{}]*",
+            min_len: 1,
+            max_len: 1000,
+        }
+    ];
 
     // guidance are instructions we give the user in case a rule fails.
-    string guidance = 6;
+    string guidance = 6 [
+        (buf.validate.field).string = {
+            pattern: "[A-Za-z][-/.!?,:;[:word:] *#\\[\\]()`{}]*",
```

Comment:
Again, I think this is markdown.  We should add to the comment in all the markdown cases, saying:

```
This value should be rendered as markdown.
```

Reply:
Removed the pattern validation 👍